code/game/objects/items/storage/toolbox.dm

```diff
@@ -84,7 +86,7 @@
 	name = "rusty blue toolbox"
 	icon_state = "toolbox_blue_old"
 	has_latches = FALSE
-	material_flags = NONE
```

Comment:
?

Reply:
It's the same as the parent's var so it's redundant to have here